packages/belos/src/BelosSolverManager.hpp

```diff
@@ -43,6 +43,90 @@
 #ifndef BELOS_SOLVERMANAGER_HPP
 #define BELOS_SOLVERMANAGER_HPP
 
+// These are here for convenience while developing the DII
+// I wanted to keep each cpp file as minimal as possible but not sure where
+// these macros could best live.
```

Comment:
`Belos::SolverManager` is Belos' main user-facing interface, so we shouldn't put implementation detail macros like this in `BelosSolverManager.hpp`.

Reply:
Now that the registration all happens in a single cpp these macros were moved there and were simplified. So this issue is resolved,